subprojects/packagefiles/win-iconv/meson.build

```diff
@@ -0,0 +1,66 @@
+project(
+  'win-iconv',
+  'c',
+  version: '0.0.8',
+  meson_version: '>=1.1.0',
+)
+
+if host_machine.system() != 'windows'
+  error('win-iconv is only supported on windows, and not supported on '
+    + host_machine.system())
+endif
+
+cc = meson.get_compiler('c')
+
+c_args = []
+
+if cc.get_argument_syntax() == 'msvc'
```

Comment:
I believe the define is needed with Windows headers, regardless of the compiler.  I'd suggest making the define unconditional since we only support Windows anyway.

Reply:
I just copied it from the original Makefile, but they might have done it incorrectly 👍🏼